### ISSUE TITLE
Fail to publish

### ISSUE THREAD

**Issue by Seeryos** [pragmaflowinc/adalo-geolocation#2]
Hi,

Thank for your sharing. I followed steps, but I needed to rename "name" field in package.json or I get error like :

```
Adalo Marketplace

  Validating package.json configuration

  Oh no, something went wrong:

  - User does not have authorization to publish new versions to pf-adalo-geo-location

  For assistance publishing please visit https://developers.adalo.com
```

Once, field value changed, I make a publish command and I get this error : 

```
./build/components/GeoFilteredList/EmptyState.js
Module not found: Error: Can't resolve './sqr-empty-state.png' in '/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList'
resolve './sqr-empty-state.png' in '/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList'
  using description file: /Users/yohan/Desktop/adalo/adalo-geolocation/package.json (relative path: ./build/components/GeoFilteredList)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /Users/yohan/Desktop/adalo/adalo-geolocation/package.json (relative path: ./build/components/GeoFilteredList/sqr-empty-state.png)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png doesn't exist
      .web.js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.js doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.json doesn't exist
      .web.jsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.jsx doesn't exist
      .jsx
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.jsx doesn't exist
      .web.react.js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.react.js doesn't exist
      .react.js
        Field 'browser' doesn't contain a valid alias configuration
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.react.js doesn't exist
      as directory
        /Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png doesn't exist
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.js]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.js]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.json]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.jsx]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.jsx]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.web.react.js]
[/Users/yohan/Desktop/adalo/adalo-geolocation/build/components/GeoFilteredList/sqr-empty-state.png.react.js]
 @ ./build/components/GeoFilteredList/EmptyState.js 158:39-71
 @ ./build/components/GeoFilteredList/GeoFilteredListComponent.js
 @ ./build/components/GeoFilteredList/index.js
 @ ./index.js
 @ ./dist/input/editor.js
```

Can you help me ? Thx

**Comment by cmdaustin** [comment]
I also had the same issue but I figured it out. One of the .png files is not being included when you build the project so I just manually copied it over from the src folder.

Go to:

`/src/components/GeoFilteredList/sqr-empty-state.png`

Copy that exact .png file. Now go back to the root folder and cd into:

`/build/components/GeoFilteredList`

Now paste that .png file in this folder then run `npx adalo publish` again
